### ISSUE TITLE
Breaking on node v11. Pls add a check for globalThis before use

### ISSUE THREAD

**Issue by tm-ssingh** [feross/queue-microtask#17]
Could you please review this issue and just add a check before using globalThis. As its not supported on node v11, my project is breaking. Thank you

**Comment by pablochud** [comment]
I have problem with that in my building process in CI pipeline. Please fix it asap.
```
globalThis is not defined
ReferenceError: globalThis is not defined
```

**Comment by tm-ssingh** [comment]
so any solution? Can the owner merge the fixes suggested by other people?

**Comment by feross** [comment]
A new version that removes `globalThis` has been released as `1.2.3`.
